src/backend/jitsi_magnify/tests/test_redirect.py

```diff
@@ -0,0 +1,63 @@
+"""
+Test suite for redirection to jitsi
+"""
+from datetime import timedelta
+from unittest import mock
+
+from django.conf import settings
+from django.test import TestCase
+from django.utils import timezone
+
+import jwt
+
+
+class RedirectTestCase(TestCase):
+    """Test suite for redirection to jitsi instance"""
+
+    def test_redirection(self):
+        """
+        Get path /api/token/test and redirect to jitsi with a token
+        We thus verify that this redirects as wanted, and that the token is correct
```

Comment:
The test is nice but philosophically not 100% satisfying.

You are placing yourself in the context of your dev environment, at the time it is now, and then you are recalculating the token with the same method as the code you are trying to test... This means that for example, if the underlying jwt library changes its behavior, you could miss a big change that could potentially break the feature without you noticing.

A better test philosophy is to define the whole context for your test and then check that the token is exactly what you expect. It's also much easier to write such test than recalculating everything like you do :wink: 

In this case you would: 
- define a test value for all settings impacting your tests thanks to the `override_settings` decorator
- choose a fix date to mock "timezone.now", not the moment the test is running like you do here
- hardcode the complete url you expect, including the encoded token and check that the request is redirected to it when you call the url.

This way you will be sure that the url works exactly how you expect and returns the correct value.

Also, you should write another test for a logged in user (self.client.login(...))